.github/workflows/dart.yml

```
@@ -134,18 +173,27 @@ jobs:
 
       - name: Deploy to Github Pages
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./docs
           force_orphan: true
-          commit_message: "docs: update documentation"
+          commit_message: "docs: update documentation and coverage"
           enable_jekyll: false
-          full_commit_message: "docs: update API documentation"
+          full_commit_message: "docs: update API documentation and coverage report"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)